design-reviews/recipes/task-design-review-template.md

```diff
@@ -0,0 +1,50 @@
+# Your Task Design Title Here (prefix with DRAFT/WIP to indicate level of completeness)
+
+> When developing a design document for a new task it should contain a detailed design proposal demonstrating how it will solve the goals outlined below.  
+> Not all tasks require a design review but when they do it is likely that there many unknowns or the solution may be more complex.
+> The design should include diagrams, pseudo code, interface contracts as needed to provide a detailed understanding of the proposal.
+
+* [Task Name](http://link-to-task-work-item)
+* [Story Name](http://link-to-story-work-item)
+* Organization: [Organization]
+* Engagement: [Engagement]
+* Customer: [Customer]
+* Authors: [Author1, Author2, etc.]
+
+## Overview / Problem Statement
+
+* Describe the task with a high-level summary.
+* Consider additional background and justification, for posterity and historical context.
+
+## Goals / In-Scope
+
+* List the specific goals for the task.
+* List a few bullet points of what this task will achieve and that are most relevant for the design review discussion.
+* This should include acceptance criteria required to meet the [definition of done](../../team-agreements/definition-of-done/).
+
+## Non-goals / Out-of-Scope
+
+* List the non-goals for the task.  
```

Comment:
Would 2nd bullet point be sufficient?

Reply:
Yeah after reading it couple of times, it seemed to be repetitive.